File: LeetCode/toLowerCase.java
/*
    709. To Lower Case
    https://leetcode.com/problems/to-lower-case/
 */

class Solution {
    public String toLowerCase(String str) {
        StringBuilder result = new StringBuilder();
        for(int i = 0; i < str.length(); i++) {
            char ch = str.charAt(i);
            if(ch >= 'A' && ch <= 'Z') {
                ch += 'a' - 'A';
            }
            result.append(ch);
        }
        return result.toString();
    }
}

/*
    Runtime: 0 ms, faster than 100.00% of Java online submissions for To Lower Case.
    Memory Usage: 34.3 MB, less than 99.80% of Java online submissions for To Lower Case.
 */

File: LeetCode/Swift/toLowerCase.swift
/*
    709. To Lower Case
    https://leetcode.com/problems/to-lower-case/
 */
 
class Solution {
    func toLowerCase(_ str: String) -> String {
        var res = "";
        for char in str {
            res.append(Character(String(char).lowercased()))
        }
        return res;
    }
}

/*
    Runtime: 0 ms, faster than 100.00% of Swift online submissions for To Lower Case.
    Memory Usage: 20.6 MB, less than 20.00% of Swift online submissions for To Lower Case.
 */
